src/app/Layouts/defaultPageWrapper.test.jsx

```diff
@@ -1,14 +1,31 @@
 import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
 import DefaultPageWrapper from './defaultPageWrapper';
-import { shouldShallowMatchSnapshot } from '../../testHelpers';
+import { ServiceContextProvider } from '../contexts/ServiceContext';
+import { ToggleContext } from '../contexts/ToggleContext';
+import { RequestContext } from '../contexts/RequestContext';
 
 describe('defaultPageWrapper', () => {
   const propsWithChildren = {
     children: <h2>Child element</h2>,
   };
 
-  shouldShallowMatchSnapshot(
```

Comment:
With this change you can delete
```
<GlobalStyle />
<ServiceWorkerContainer />
<ManifestContainer />
<MPulseBeacon />
```
In `simorgh/src/app/Layouts/defaultPageWrapper.jsx` and it still passes, can you add assertion tests to ensure we maintain coverage please?